### ISSUE TITLE
Make check options configurable

### ISSUE THREAD

**Issue by sschuberth** [grundprinzip/gerrit-check#6]
There should be an easy way, e.g. via an external file, to configure the options / command line arguments passed to the individual checkers.

**Comment by SiElHocine** [comment]
@sschuberth Do you have any suggestion for the implementation ? I can made the changes, I need this option too.

**Comment by sschuberth** [comment]
Not really... maybe some YAML file, which is by default looked for at `~/.config/gerrit-check.yaml`, and which simply contains key / value pairs for tool name / options?
